### PR TITLE
Create daily, weekly pipelines

### PR DESCRIPTION
**Pull request by blaisedias**

### CODE REVIEW

File: pipelines/ubuntu-20.04-lts/k8s1.21/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    parameters {
+        booleanParam(defaultValue: false, name: 'build_images', description: 'build images and then test.')
+        string(defaultValue: '', name: 'image_tag', trim: true, description: 'image tag, auto-generated if build_images is set and this field is empty')
+    }
+    stages {
+        stage('RunTestJob') {
+            steps {
+                script {
+                    // k8s version, build_images flag, image_tag
+                    job_params = Ubuntu().GetDefaultJobParameters( "1.21.8", params.build_images, params.image_tag)
```
Comment (rohan2794) [inline pipelines/ubuntu-20.04-lts/k8s1.21/Jenkinsfile:28]: prefix job_params with def keyword

Comment (chriswldenyer) [inline pipelines/ubuntu-20.04-lts/k8s1.21/Jenkinsfile:28]: maybe make this "def job_params" for consistency although this job isn't "multi-threaded"

**Review by chriswldenyer — COMMENTED** [review]
Cool! LGTM.
In the long term we should think about splitting up common.groovy, it is getting near 1000 lines which would be my limit normally.

**Review by chriswldenyer — APPROVED** [review]

File: pipelines/ubuntu-20.04-lts/k8s1.23/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    parameters {
+        booleanParam(defaultValue: false, name: 'build_images', description: 'build images and then test.')
+        string(defaultValue: '', name: 'image_tag', trim: true, description: 'image tag, auto-generated if build_images is set and this field is empty')
+    }
+    stages {
+        stage('RunTestJob') {
+            steps {
+                script {
+                    // k8s version, build_images flag, image_tag
+                    job_params = Ubuntu().GetDefaultJobParameters( "1.23.1", params.build_images, params.image_tag)
```
Comment (rohan2794) [inline pipelines/ubuntu-20.04-lts/k8s1.23/Jenkinsfile:28]: as previous comment

**Review by rohan2794 — APPROVED** [review]
LGTM, some minor comments

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Pull request merged**